sp/src/game/client/mapbase/c_trigger_portal.cpp

```diff
@@ -1,4 +1,4 @@
-//========= Copyright Valve Corporation, All rights reserved. ============//
+//================== Mapbase - Source 2013, Author(s) Krispy ===================//
```

Comment:
This is the wrong header formatting. "Mapbase - Source 2013" should be followed by the link to the repo while the "Author(s)" part should be below "Purpose:".

For example, see how I formatted the headers in reductor's VScript implementation files:
https://github.com/mapbase-source/source-sdk-2013/blob/master/sp/src/vscript/vscript.cpp

I will change the contribution guidelines to make the correct formatting more clear. Sorry for the confusion.